src/Resources/config/api_form_doctrine_mongodb.php

```diff
@@ -39,7 +39,6 @@
                 new ReferenceConfigurator('sonata.media.pool'),
                 '%sonata.media.admin.media.entity%',
             ])
-            // NEXT_MAJOR: make symfony/monolog-bundle a require dependency and remove nullOnInvalid
```

Comment:
I don't think we should add `symfony/monolog-bundle` as a required dependency. On SonataAdminBundle we didn't do it.